### PR TITLE
feat: add relay count to IndexBlock

### PR DESCRIPTION
**Pull request by crisog**
This calculates all the relay rewards claimed on a block while indexing.

### CODE REVIEW

File: indexer.go [hunk 1]
```diff
@@ -169,15 +156,57 @@ func (i *Indexer) IndexBlock(blockHeight int) error {
...
+
+	if err != nil {
+		return err
+	}
+
+	relayCount := 0
+
+	for _, tx := range blockTxs {
+		if tx.TxResult.MessageType == "claim" {
+			proofs := tx.StdTx.Msg.Value["total_proofs"].(int)
```
Comment (pablocampogo) [inline indexer.go:169]: You should verify with an ok to see if the value was an int and see how you manage it when it isn't or it doesn't exist. Example:
https://github.com/pokt-foundation/pocket-indexer-lib/blob/master/indexer.go#:~:text=rawToAddress%2C%20ok,%7D

Comment (pablocampogo) [reply indexer.go:169]: Also are we sure this comes as an INT?

File: indexer.go [hunk 2]
```diff
@@ -169,15 +156,57 @@ func (i *Indexer) IndexBlock(blockHeight int) error {
...
+	err = i.writer.WriteBlock(block)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func convertProviderBlockToBlock(providerBlock *provider.GetBlockOutput) *Block {
+func getProviderBlockTransactions(blockHeight int, indexerProvider Provider) ([]*provider.Transaction, error) {
```
Comment (pablocampogo) [inline indexer.go:184]: This function should be a method of `Indexer`, in that way you don't need to pass provider as a param

**Review by pablocampogo — CHANGES_REQUESTED** [review]

**Comment by crisog** [comment]
Closing in favor of a different approach

**Pull request closed without merging**